backend/src/router.js

```diff
@@ -29,6 +30,14 @@ router.put("/products/:id", ProductController.edit);
 router.post("/products", ProductController.add);
 router.delete("/products/:id", ProductController.delete);
 
+router.get("/affiliations", AffiliationController.browse);
+router.get("/affiliations/products", AffiliationController.readProducts);
```

Comment:
what is the link between affiliations and products ? following rest conventions, this should be `/affiliations/:id/products` or just `/products`